src/app/projects/components/project-settings/helper/label-helper.ts

```diff
@@ -75,8 +75,14 @@ export class LabelHelper {
 
         labelInput.value = '';
         labelInput.focus();
-        this.settings.requestTimeOut = true;
-        timer(100).subscribe(() => this.settings.requestTimeOut = false);
+
+        let subject = new Subject<boolean>();
```

Comment:
is there a specific reason why this is now with a subscription solved? using an object reference as stated before should do the same without the need to handle the subscriptions in any specific way. Also the subscription seems to be open indefinitely though it is only used once.